src/test/java/collab/EmployeeManagerTest.java

```diff
@@ -0,0 +1,154 @@
+        assertTrue(((AddCommand) commandList.get(0)).getFirstOption() instanceof NoneFirstOption);
+        assertTrue(((AddCommand) commandList.get(0)).getSecondOption() instanceof NoneSecondOption);
+        assertTrue(((AddCommand) commandList.get(0)).getThirdOption() instanceof NoneThirdOption);
+
+        assertTrue(commandList.get(1) instanceof SearchCommand);
+        assertTrue(((SearchCommand) commandList.get(1)).getFirstOption() instanceof PrintOption);
+        assertTrue(((SearchCommand) commandList.get(1)).getSecondOption() instanceof DayOfBirthdayOption);
+        assertTrue(((SearchCommand) commandList.get(1)).getThirdOption() instanceof NoneThirdOption);
+
+        assertTrue(commandList.get(2) instanceof ModifyCommand);
+        assertTrue(((ModifyCommand) commandList.get(2)).getFirstOption() instanceof PrintOption);
+        assertTrue(((ModifyCommand) commandList.get(2)).getSecondOption() instanceof EmptySecondOption);
+        assertTrue(((ModifyCommand) commandList.get(2)).getThirdOption() instanceof NoneThirdOption);
+
+        assertTrue(commandList.get(3) instanceof DeleteCommand);
+        assertTrue(((DeleteCommand) commandList.get(3)).getFirstOption() instanceof PrintOption);
+        assertTrue(((DeleteCommand) commandList.get(3)).getSecondOption() instanceof LastNameOption);
+        assertTrue(((DeleteCommand) commandList.get(3)).getThirdOption() instanceof NoneThirdOption);
+    }
+
+    @Test
+    public void executeCommandTest() throws Exception{
+        EmployeeManager employeeManager = new EmployeeManager();
+        List<ICommand> commandList = employeeManager.parseCommandList(commandStringList1);
+        String executionResult = employeeManager.executeCommandList(commandList);
+        String[] resultLines = executionResult.split("\n");
+        assertTrue(resultLines.length == 3);
+        resultLines[0].equals("SCH,18117906,TWU QSOLT,CL4,010-6672-7186,20030413,PRO");
+        resultLines[1].equals("MOD,18117906,TWU QSOLT,CL3,010-6672-7186,20030413,PRO");
+        resultLines[2].equals("DEL,18117906,TWU QSOLT,CL3,010-6672-7186,20030413,PRO");
+    }
+
+    @Test
+    void saveResultToFileTest() throws Exception {
+        assertTrue(!new File(employeeManagerSaveFileTestFilePath).exists());
+        EmployeeManager employeeManager = new EmployeeManager();
+        List<ICommand> commandList = employeeManager.parseCommandList(commandStringList1);
+        String executionResult = employeeManager.executeCommandList(commandList);
+        employeeManager.saveExecutionResultToFile(employeeManagerSaveFileTestFilePath, executionResult);
+
+        assertTrue(new File(employeeManagerSaveFileTestFilePath).exists());
+        ArrayList<String> contents = new ArrayList<String>();
+        BufferedReader br = new BufferedReader(new FileReader(employeeManagerSaveFileTestFilePath));
+        String str;
+        while ((str = br.readLine()) != null) {
+            contents.add(str);
+        }
+        br.close();
```

Comment:
close()를 빼 먹을 우려가 있어서 아래와 같이 변경하는 것은 어떨까요?

```suggestion
        try(FileReader fr = new FileReader(employeeManagerSaveFileTestFilePath); BufferedReader br = new BufferedReader(fr)) {
            String str;
            while ((str = br.readLine()) != null) {
                contents.add(str);
            }
        } catch(Exception e) {
            throw e;
        }
```

Reply:
아네 throws Exception 처리가 되어 있기는한데, 다시한번 확인해 보겠습니다.